### PR TITLE
Update tutorial-console.md

### PR DESCRIPTION
**Pull request by ghost**
Add information about the console helper class.

### CODE REVIEW

File: docs/guide/tutorial-console.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ It's a good practice to define meaningful constants for your controller in case
 
 ### Formatting and colors
 
-Yii console supports formatted output that is automatically degraded to non-formatted one if it's not supported
+The [yii\helper\Console](structure-assets.md#using-the-asset-command) supports formatted output that is automatically degraded to non-formatted one if it's not supported
```
Comment (cebe) [inline docs/guide/tutorial-console.md]: why do you link to structue-assets?

Comment (llfm) [inline docs/guide/tutorial-console.md]: LOL, fixed.

**Comment by cebe** [comment]
well, this is not really the right place, the sentence should stay as is imo but we could give links to stdout,stderr and ansiFormat methods as well as linking to the console helper constants.

**Comment by SilverFire** [comment]
Closed by https://github.com/yiisoft/yii2/commit/f7cf6416353cf1237662043c15a2dde4165e665e
Thank you

**Pull request closed without merging**